src/material-experimental/mdc-chips/harness/chip-input.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ComponentHarness} from '@angular/cdk-experimental/testing';
+
+/**
+ * Harness for interacting with a grid's chip input in tests.
+ * @dynamic
+ */
+export class MatChipInputHarness extends ComponentHarness {
+  static hostSelector = 'input.mat-mdc-chip-input';
```

Comment:
Is the `input` part necessary?

Reply:
No, can't remember why I included that but it doesn't seem necessary